app/src/main/java/com/github/leuludyha/ibdb/presentation/screen/auth/signup/SignUpScreenViewModel.kt

```diff
@@ -21,26 +23,63 @@ class SignUpScreenViewModel @Inject constructor(
      * Saves a boolean which indicates that the walk-through has been completed once
      * by the user. The boolean is persisted through sessions (Application restarts)
      */
-    fun rememberWalkThroughIsCompleted() {
+    fun rememberWalkThroughIsCompleted(context: Context) {
+        val sp = context.getSharedPreferences(
+            Constant.SIGN_UP_SHARED_PREFERENCES_KEY,
+            Context.MODE_PRIVATE
```

Comment:
What does `MODE_PRIVATE` do ?

Reply:
It prevents other applications from accessing these stored values.